services/httpd/response_writer.go

```diff
@@ -25,19 +25,27 @@ type ResponseWriter interface {
 func NewResponseWriter(w http.ResponseWriter, r *http.Request) ResponseWriter {
 	pretty := r.URL.Query().Get("pretty") == "true"
 	rw := &responseWriter{ResponseWriter: w}
-	switch r.Header.Get("Accept") {
-	case "application/csv", "text/csv":
-		w.Header().Add("Content-Type", "text/csv")
-		rw.formatter = &csvFormatter{statementID: -1}
-	case "application/x-msgpack":
-		w.Header().Add("Content-Type", "application/x-msgpack")
-		rw.formatter = &msgpackFormatter{}
-	case "application/json":
-		fallthrough
-	default:
-		w.Header().Add("Content-Type", "application/json")
-		rw.formatter = &jsonFormatter{Pretty: pretty}
+
+	acceptHeaders := parseAccept(r.Header.Get("Accept"))
```

Comment:
Can you change this to `r.Header` as the argument and change `parseAccept` to take a `[]string` value? I think it's possible for someone to do this:

```
Accept: application/csv
Accept: application/x-msgpack
```